rllib/algorithms/algorithm_config.py

```diff
@@ -600,8 +600,10 @@ def update_from_dict(
             # correct methods to properly `.update()` those from given config dict
             # (to not lose any sub-keys).
             elif key == "callbacks_class":
-                # Resolve possible classpath.
-                value = deserialize_type(value, error=True)
+                # For backward compatibility reasons, only resolve possible
+                # classpath if value is a str type.
+                if isinstance(value, str):
```

Comment:
Should we just do `error=False` here instead of the if block?
This way, if value is already a class, nothing will happen.